packages/admin/src/assets/examples/events_appointment.csv

```diff
@@ -0,0 +1 @@
+startedAt,endedAt,personId,resourceId
```

Comment:
What is the purpose of all these csv files with only headers?
Also, being in the asset folder will mean these get deployed.  Is that intended?

Reply:
these are the sample file that get downloaded when you press download example. they are going to be needed in the app.

Reply:
I would prefer these get generated, so we can make them dynamic in the future.  We will want to support dimension columns which will be data based.  If we want to keep them for now, that's fine with me.  Do we have a story to support importing with dimensions?  If, so I'd like dynamic file generations be a requirement in that.  If not, would you make one or update a confluence page.

Reply:
i agree, i think the fianl version should have a generated solution, i will add that info to the ticket or make a new one.